### PR TITLE
T-GM tests: Validate the signal coming from the GNSS

### PR DESCRIPTION
**Pull request by sabinaaledort**
Add a test to verify that a valid 1PPS signal coming from the GNSS receiver arrives at the DPLL of the WPC card.

### CODE REVIEW

**Comment by sabinaaledort** [comment]
/hold
needs a rebase after merging https://github.com/redhat-eets/sno-tests/pull/8

**Review by jnunyez — COMMENTED** [review]
@sabinaaledort The procedure is ok for getting the 1PPS signal coming from GNSS. But the reported phase offset is wrong due to bug in ice OOT 1.9.11. There is a new method to calculate this in ICE OOT 1.10.1.2. Since I read SMB is interested on DPLL measurement I would suggest you to upgrade ICE OOT 1.10.1.2.

**Comment by jnunyez** [comment]
@sabinaaledort Also be aware of the clonficts before merging.

**Comment by crwr45** [comment]
Looks good once the previous PR is merged and this is rebased.

File: test/pkg/consts/const.go
Comment (jnunyez) [inline test/pkg/consts/const.go:7]: Exact device type is E810XXVDA4T. See suggestion below:

```suggestion
 	WPCDeviceType = "E810-XXVDA4T"
```

File: test/pkg/consts/const.go
Comment (jnunyez) [inline test/pkg/consts/const.go:10]:
```suggestion
        // State of DPLL should be DPLL Locked Holdover Acquired
	DPLLLockedHOACQState           = "3"
```

**Review by jnunyez — CHANGES_REQUESTED** [review]
See suggestions above.

File: test/pkg/consts/const.go
Comment (sabinaaledort) [reply test/pkg/consts/const.go:7]: The device type we have is `E810-XXV-4T` do you have `E810-XXVDA4T`? Also that was part of the previous PR, not set in this PR, so we can open another PR to change it or that will be done in a PR that's adding the config yaml you previously suggested.

Comment (jnunyez) [reply test/pkg/consts/const.go:7]: let's do that later then :-)

**Pull request merged**
